index.html

```diff
@@ -8,10 +8,10 @@
 <body>
   <h1>Pixel Art!</h1>
   <div class="controls">
-    <form id="form">
+
```

Comment:
heh heh.  Tricky, tricky.  Did you do this to keep the page from refreshing when you clicked on the link?  You can also use `event.preventDefault()` to do that, as we discussed in the readme.